specs/mediawiki/v1/content.yaml

```diff
@@ -558,6 +582,21 @@ definitions:
         type: string
       uri:
         type: string
+  httpError:
+    required:
+      - method
```

Comment:
I wonder if we should move 'method' down into a req sub-object that has just the uri & method.